src/sections/app.style.js

```diff
@@ -43,6 +43,7 @@ const GlobalStyle = createGlobalStyle`
   }
 
 html{
+    overflow-x: hidden;
```

Comment:
have you tested this all pages and its not having negative effect right?

Reply:
yes I have tested all pages , on my side there are no negative effects